src/Snippets/Token/DeleteTrackTokenSnippet.php

```diff
@@ -47,9 +47,9 @@ class DeleteTrackTokenSnippet extends ChangeReceptionCodeSnippetAbstract
     protected ?string $_replacementTokenId;
 
     /**
-     * @var Token
+     * @var Token|null
      */
-    protected Token $_replacementToken;
+    protected ?Token $_replacementToken = null;
```

Comment:
Deze snippet werkt niet zonder deze wijziging. Mij lijkt deze wijziging redelijk, maar ben niet zeker of het inderdaad correct is.